### DISCUSSION TITLE
cp30开票测试

### DISCUSSION THREAD

**Discussion by Hxy1899** [mikumifa/biliTickerBuy#431] [提问专区] [answered]
小芒平台票务测试，首先填写实名信息，然后关于着装登记是否cos（在备注一栏，需要填写是/否），不填提交不了，然后才能付款。不知道另外平台会不会也做这样处理

**Comment by mikumifa** [comment]
B站有专门登记的地方（bw就是，不用担心

**Comment by gytguxixin** [comment]
呃呃这个bilibili的抢票网站没法识别cpp的网址啊（
这个怎么解决啊

**Reply by GGuo555** [reply, marked as answer]
去看作者的另一个repo cpptickerbuy
